internal/nostr/nostr.go

```diff
@@ -724,98 +756,6 @@ func (svc *Service) getRelayConnection(ctx context.Context, customRelayURL strin
 	}
 }
 
-func (svc *Service) processRequest(ctx context.Context, subscription *Subscription, requestEvent *RequestEvent, requestData *NIP47Request) (*nostr.Event, int, error) {
-	publishState := REQUEST_EVENT_PUBLISH_FAILED
-	defer func() {
-		subscription.Open = false
-		requestEvent.State = publishState
-		svc.db.Save(subscription)
-		svc.db.Save(requestEvent)
-	}()
-	relay, isCustomRelay, err := svc.getRelayConnection(ctx, subscription.RelayUrl)
```

Comment:
Marking here: the `isCustomRelay` logic looks quite dangerous, it disconnects the relay after each request and in multiple places. If there are two simultaneous requests to the same custom relay, this could break